azure/v28.0.0/release.diff

```diff
@@ -0,0 +1,118 @@
+apiVersion: release.giantswarm.io/v1alpha1				apiVersion: release.giantswarm.io/v1alpha1
```

Comment:
Diff looks strange

Reply:
Scratch that – it's a side-by-side which looks really nasty on a small screen with the GitHub PR open

Reply:
How exactly does it look strange for you? Actually I only did `diff <old> <new>` and I agree, depending on how you're reading it, tabs get display correctly or not. It sucks... but still better than nothing.